#service for gettin model_states from gazebo

std_msgs/Empty empty
---
last_letter_2/Model_states model_states